hieradata/development.yaml

```diff
@@ -151,7 +151,7 @@ govuk::apps::email_alert_api::db_password: "%{hiera('govuk::apps::email_alert_ap
 govuk::apps::email_alert_api::db_port: 6432
 govuk::apps::email_alert_api::db_allow_prepared_statements: false
 # This list should be kept in sync with https://www.notifications.service.gov.uk/services/ecfc7e2f-5145-45a6-9413-5d9b6e813ea9/users
-govuk::apps::email_alert_api::email_address_override_whitelist:
+govuk::apps::email_alert_api::email_address_override_allowlist:
```

Comment:
At least to me, `allowlist` doesn't make as much sense. I'm fine with changing this, but how about `allowed_email_addresses_override`?

Reply:
Or maybe just `allowed_email_addresses`, I'm not sure what the override bit means.

Reply:
I like your suggestion!